samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Bots/TeamsMessagingExtensionsActionBot.cs

```diff
@@ -21,15 +23,50 @@ protected override async Task<MessagingExtensionActionResponse> OnTeamsMessaging
         {
             switch (action.CommandId)
             {
-                // These commandIds are defined in the Teams App Manifest.
                 case "createCard":
                     return CreateCardCommand(turnContext, action);
-
                 case "shareMessage":
                     return ShareMessageCommand(turnContext, action);
-                default:
-                    return await Task.FromResult(new MessagingExtensionActionResponse());
+                case "webView":
```

Comment:
I'm confused about this... the command name is "webView" but, the response is a ThumbnailCard?

Same with HTML, and 'RazorViewCard' <-- these are not returning html, or razor views, but cards.

Reply:
Task Module load s the HTML and razor page  but when we submit the response it goes as thumbnail or adaptive card in compose box

Reply:
I've given the meaningful names to those methods